### PR TITLE
Adds release notes for upcoming 1.6 release

### PR DESCRIPTION
**Pull request by fredlf**
Signed-off-by: Fred Lifton <fred.lifton@docker.com>

### CODE REVIEW

**Comment by ehazlett** [comment]
LGTM

Holding until release so these do not get pushed early.

File: docs/index.md [hunk 1]
```diff
@@ -924,3 +924,22 @@ Options:
...
 The VMware vSphere driver uses the latest boot2docker image.
+
+## Release Notes
+
+### Version 0.2.0 (April 16, 2015)
+
+For complete information on this release, see the [0.2.0 Milestone project page](https://github.com/docker/machine/wiki/0.2.0-Milestone-Project-Page).
+In addition to bug fixes and refinements, this release adds the following:
+
+* Updated and refactored Driver interface For details, see
```
Comment (nathanleclaire) [inline docs/index.md]: missing `.` here

**Comment by nathanleclaire** [comment]
Other than a very small comment, LGTM thanks Fred

**Comment by nathanleclaire** [comment]
:+1:

**Pull request merged**